packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -220,6 +220,20 @@ export class Account {
   ): Promise<AccountHistory[]> {
     return await this.client.call('listaccounthistory', [owner, options], 'number')
   }
+
+  /**
+   * Creates (and submits to local node and network) a transfer transaction from the wallet UTXOs to specfied account.
+   * The second optional argument (may be empty array) is an array of specific UTXOs to spend.
+   *
+   * @param {any} [payload]
```

Comment:
that should be not `any`

Reply:
The value can be 

{address1: 1DFI}
{address1: 1DFI, address2: 2DFI}
{address1: 1DFI, address2: 2DFI, address3: 3DFI}

All are objects. I just can't think of another type other than any.

Reply:
try look at this
https://stackoverflow.com/questions/39256682/interface-for-dynamic-key-in-typescript